src/statistics/StatisticalApi.ts

```diff
@@ -0,0 +1,71 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { RequestConfiguration } from '..';
+import { getAuthToken } from '../auth';
+import { GetStatsParams } from '../layer/const';
+import { CACHE_CONFIG_30MIN } from '../utils/cacheHandlers';
+import { getAxiosReqParams } from '../utils/cancelRequests';
+import { StatisticsProvider } from './StatisticsProvider';
+import { AbstractSentinelHubV3Layer } from '../layer/AbstractSentinelHubV3Layer';
+import { StatisticsUtils } from './statistics.utils';
+
+import { StatisticalApiResponse, StatisticalApiPayload } from './const';
+
+const STATS_DEFAULT_OUTPUT = 'default';
+
+export class StatisticalApi implements StatisticsProvider {
+  public async getStats(
+    layer: AbstractSentinelHubV3Layer,
+    params: GetStatsParams,
+    reqConfig?: RequestConfiguration,
+  ): Promise<StatisticalApiResponse> {
+    const authToken = reqConfig && reqConfig.authToken ? reqConfig.authToken : getAuthToken();
+    if (!authToken) {
+      throw new Error('Must be authenticated to use Statistical API');
+    }
+
+    await layer.updateLayerFromServiceIfNeeded(reqConfig);
+
+    const input = await StatisticsUtils.createInputPayload(layer, params, reqConfig);
+    const aggregation = StatisticsUtils.createAggregationPayload(layer, {
+      ...params,
+      aggregationInterval: 'P1D',
```

Comment:
If I understand this correctly, the `aggregationInterval` is hardcoded to one day? 

I guess it's indeed best  if we just implement what we need for EOB and then we can deal with full support of all Statistical API parameters later (if required).